### ISSUE TITLE
3.0 Release in NPM is not compiled to Javascript

### ISSUE THREAD

**Issue by orthanc** [amplify-education/serverless-log-forwarding#70]
The [3.0.0 release currently published in NPM](https://www.npmjs.com/package/serverless-log-forwarding/v/3.0.0) does not seem to work with Serverless 2.72 (I suspect but have not tested that it also doesn't work with Serverless 3, at least without some additional plugins)

When using it, I get the following error:

> Serverless plugin “serverless-log-forwarding” not found. Make sure it’s installed and listed in the “plugins” section of your serverless config file. Run “serverless plugin install -n serverless-log-forwarding” to install it.

The [2.0.0 release](https://www.npmjs.com/package/serverless-log-forwarding/v/2.0.0) works as expected.

Inspecting the `node_modules/serverless-log-forwarding` directory after installing 3.x, it appears the source of the problem is that the 3.0.0 release contains the typescript source files, not the compiled javascript.

Specifically, the Contents of the TAR from NPM are
```
package/.eslintignore
package/.eslintrc
package/LICENSE
package/test/integration-tests/fixtures/resources/logs-receiver.js
package/package.json
package/tsconfig.eslint.json
package/tsconfig.json
package/CHANGELOG.md
package/README.md
package/test/integration-tests/configs/create-perm-off/logs_producer.py
package/test/integration-tests/configs/multiple-one-disabled/logs_producer.py
package/test/integration-tests/configs/multiple-producers/logs_producer.py
package/test/integration-tests/configs/single-filter/logs_producer.py
package/test/integration-tests/configs/single-producer/logs_producer.py
package/test/integration-tests/configs/stage-match/logs_producer.py
package/test/integration-tests/configs/stage-no-match/logs_producer.py
package/test/integration-tests/base.ts
package/test/integration-tests/fixtures/custom-policy.ts
package/test/unit-tests/index-test.ts
package/src/index.ts
package/test/integration-tests/integration-tests.ts
package/test/integration-tests/fixtures/logs-receiver.ts
package/test/integration-tests/test-utilities.ts
package/src/types.ts
package/test/integration-tests/types.ts
package/test/integration-tests/configs/create-perm-off/serverless.yml
package/test/integration-tests/configs/multiple-one-disabled/serverless.yml
package/test/integration-tests/configs/multiple-producers/serverless.yml
package/test/integration-tests/configs/single-filter/serverless.yml
package/test/integration-tests/configs/single-producer/serverless.yml
package/test/integration-tests/configs/stage-match/serverless.yml
package/test/integration-tests/configs/stage-no-match/serverless.yml
package/.github/workflows/test-publish.yml
package/test/integration-tests/fixtures/resources/logs-receiver.zip
```

Notice that while there is a `src` and `test` directory, there is no `dist` directory. But the `package.json` refers to `dist/src/index.js` which I presume is intended to be the compiled javascript:

```
"main": "dist/src/index.js"
```

As a result, the NPM module doesn't work without compilation so can't be used.

**Comment by rddimon** [comment]
Hi @orthanc 

Thank you for your support!

We have released a fix in version `3.0.1`
https://github.com/amplify-education/serverless-log-forwarding/pull/71

And I was wondering if you could try it out and let us know!

**Comment by orthanc** [comment]
Sorry for the delayed reply, we've updated and `3.0.1` works fine for us.

Thank you for the quick response.
